en/asgardeo/docs/references/service-extensions/in-flow-extensions/custom-authentication/api/custom-authentication-action-v1.yaml

```diff
@@ -428,8 +432,8 @@ components:
           format: uri
           description: Claim uri defined in the 'http://wso2.org/claims' dialect
         value:
-          type: string
-          description: Claim value
+          type: object
+          description: Claim value. If the claim is a multi-valued claim, it should be a list of strings; otherwise, it should be a string.
```

Comment:
I think we should also mention that the 'MultiAttributeSeparator' is not expected in data of a multi valued claim.

You will have to mention this differently for IS and Asgardeo as this 'MultiAttributeSeparator' is a server level config. In Asgardeo it becomes an internal keyword used